pkg/stub/handler.go

```diff
@@ -606,18 +638,66 @@ func (h *Handler) CleanUpOpenshiftNamespaceOnDelete(srcfg *v1alpha1.SamplesResou
 			err = h.templateclientwrapper.Delete("openshift", temp.Name, &metav1.DeleteOptions{})
 			if err != nil && !kerrors.IsNotFound(err) {
 				logrus.Warnf("Problem deleting openshift template %s on SamplesResource delete: %#v", temp.Name, err)
+			} else {
+				err = nil
```

Comment:
what's the point of this?  you don't do anything with this value.

Reply:
ah leftover ... removing